server/util/clickhouse/schema/schema.go

```diff
@@ -192,6 +192,10 @@ type Execution struct {
 	CachedResult bool
 	DoNotCache   bool
 
+	// Long string fields
+	CommandSnippet string
```

Comment:
need to remove these from ExcludedFields()

Reply:
done